src/image.rs

```diff
@@ -169,7 +169,7 @@ pub fn pull_image(name: &str) -> Result<(), Error> {
 /// Determine whether an image exists locally
 pub fn image_exists_locally(name: &str) -> Result<bool, Error> {
     let ret = Command::new("docker")
-        .args(&["history", "docker:stable-dind"])
+        .args(&["history", "docker:dind"])
```

Comment:
This isn't your fault, but this procedure looks like its doing totally the wrong thing (the `name` parameter is being totally ignored in the actual command). This is only used in the `dind` source file to check for the `dind` image, but will be wrong when we specify a custom or pinned version - could you either generalise this procedure to use the name parameter properly?

Reply:
I've updated to use the name parameter properly and added a log for the name that is being checked.